model/LGCN-TensorFlow/utility/load_data.py

```diff
@@ -208,9 +273,9 @@ def negative_pool(self):
 
     def sample(self):
         if self.batch_size <= self.n_users:
-            users = rd.sample(population=list(self.exist_users_train), k=self.batch_size)
+            users = rd.sample(list(self.exist_users), self.batch_size)
```

Comment:
der er ikke chance for at vi tager en user der kun er i test her?

Reply:
Nej, test users og test items er et subset af train items of users